django/db/models/aggregates.py

```diff
@@ -20,6 +20,11 @@ def __init__(self, *args, filter=None, **kwargs):
         self.filter = filter
         super().__init__(*args, **kwargs)
 
+    def __neg__(self):
+        if hasattr(self, 'function'):
+            self.function = '-' + self.function
+        return self
```

Comment:
1. I'm not sure this is the best way to do it - `return self * Value(-1)` is probably cleaner and more general
2. The current structure really only negates *if `self.function` is set*, otherwise it silently *does not* negate. I can't see the case where a concrete aggregate class does not have `self.function`, but it's theoretically possible that a third-party aggregate out there doesn't use it. In general, this code structure is very fragile; it's much better to error than to silently do the wrong thing.

Reply:
@charettes suggested the `* -1` on the ticket as a current workaround. Also the explicit `Value()` above isn't necessary since `Func` will automatically wrap `-1`.

Reply:
> I have added an explicit default_alias for the result if the original class (the one being negated) specifies a default_alias.

I'm not certain we should be doing that, and if we move `__neg__` to `Combinable` as suggested below, it wouldn't make sense.